### PR TITLE
Integrating toolkit into samples repo

### PR DESCRIPTION
**Pull request by dg0yal**
This is in prep for the upcoming AR samples that will need the toolkit.

The toolkit repo has been added as a submodule. This will help users who use the installed sdk.

For those using cocoapods, a podfile has been added.

Note - We will need to update the podfile and the submodule to point to 100.6 toolkit when it is released.

Currently I am referencing 100.5 version of the toolkit.

@mhdostal @philium

### CODE REVIEW

File: podfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
 platform :ios, '11.0'
 target 'ArcGIS Runtime SDK Samples' do
-  pod 'ArcGIS-Runtime-SDK-iOS', '~> 100.6'
+  pod 'ArcGIS-Runtime-Toolkit-iOS', '~> 100.5'
```
Comment (mhdostal) [inline podfile]: This will need to be updated to 100.6 once the Toolkit is released.

Comment (dg0yal) [reply podfile]: Yup

Comment (dg0yal) [reply podfile]: And also the submodule reference

**Comment by dg0yal** [comment]
@philium done

**Pull request merged**